opal/mca/reachable/netlink/reachable_netlink_module.c

```diff
@@ -112,6 +112,15 @@ static int get_weights(opal_if_t *local_if, opal_if_t *remote_if)
         remote_ip = (uint32_t)((struct sockaddr_in *)&(remote_if->if_addr))->sin_addr.s_addr;
         outgoing_interface = local_if->if_kernel_index;
 
+        /* If the ips are identical, assume reachable through loopback */
```

Comment:
Can you add a bit more of a comment here?  Say that we're doing something artificial because historical reasons.  Someone in 2 years is going to wonder why this comment is here.

Reply:
I updated the comment.